service/controller/v5/resource/blobobject/resource.go

```diff
@@ -0,0 +1,197 @@
+package blobobject
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"strings"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2018-07-01/storage"
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	"github.com/giantswarm/azure-operator/client"
+	"github.com/giantswarm/azure-operator/service/controller/v5/cloudconfig"
+	"github.com/giantswarm/azure-operator/service/controller/v5/controllercontext"
+)
+
+const (
+	blobFormatString      = `https://%s.blob.core.windows.net`
+	resourceNotFoundError = "NotFound"
+	maxRetriesRequests    = 3
+	// Name is the identifier of the resource.
+	Name = "blobobjectv5"
+)
+
+type Config struct {
+	HostAzureClientSetConfig client.AzureClientSetConfig
+	Logger                   micrologger.Logger
+}
+
+type Resource struct {
+	hostAzureClientSetConfig client.AzureClientSetConfig
+	logger                   micrologger.Logger
+}
+
+type StorageClient struct {
+	accountsClient *storage.AccountsClient
+	containerURL   azblob.ContainerURL
+}
```

Comment:
Can this be moved to `storage_client.go`? It would be easier to distinguish what is where. Now all is intertwined.